pilot/pkg/xds/eds_test.go

```diff
@@ -186,7 +186,7 @@ func TestSAUpdate(t *testing.T) {
 		Service:     svc,
 		ServicePort: svc.Ports[0],
 		Endpoint: &model.IstioEndpoint{
-			Address:        "1.2.3.4",
+			Addresses:      []string{"1.2.3.4"},
```

Comment:
No dual-stack tests in this file too

Reply:
done, add a new unit test `TestSAUpdateWithMulAddrsInstance` for multiple addresses unit test.